feeds/scheduler/feed_group.go

```diff
@@ -61,13 +61,11 @@ func (fg *FeedGroup) pollAndPublish() groupResult {
 		return result
 	}
 	log.WithField("num_packages", len(pkgs)).Printf("Publishing packages...")
-	numPublished, pubErr := fg.publishPackages(pkgs)
-	result.numPublished = numPublished
-	if pubErr != nil {
-		log.Errorf("Failed to publish %v packages due to err: %v", len(pkgs)-numPublished, pubErr)
-		result.pubErr = errPub
-	} else {
-		log.WithField("num_packages", numPublished).Printf("Successfully published packages")
+	numPublished, err := fg.publishPackages(pkgs)
```

Comment:
```suggestion
	result.numPublished, result.pubErr := fg.publishPackages(pkgs)
```
NIT: Does this work?

Reply:
You can't do a shorthand `:=` declaration to a predefined struct's field (where as you can to a generic var, see https://gist.github.com/chmike/ac0113afefbc04e67323b4a3688d6b54#-cant-be-used-with-member-fields). However in this case a normal `=` declaration is fine as we're casting both values to the predefined struct so thanks!